akka-actor-typed/src/main/scala/akka/actor/typed/scaladsl/ActorContext.scala

```diff
@@ -129,6 +129,16 @@ trait ActorContext[T] extends TypedActorContext[T] with ClassicActorContextProvi
    */
   def spawn[U](behavior: Behavior[U], name: String, props: Props = Props.empty): ActorRef[U]
 
+  /**
+   * Delegate message and signal's execution by given [[akka.actor.typed.Behavior]]
+   * using [[Behavior.interpretMessage]] or [[Behavior.interpretSignal]]
+   *
+   * note: if given [[akka.actor.typed.Behavior]] resulting [[Behaviors.same]] that will cause context switching to the given behavior
+   * and if result is [[Behaviors.unhandled]] that will trigger the [[akka.actor.typed.scaladsl.ActorContext.onUnhandled]]
+   * then switching to the given behavior.
+   */
+  def delegate(delegator: Behavior[T], msg: T): Behavior[T]
```

Comment:
probably need an exclusion rule in the mima filters for the binary compatibility check

Reply:
added